pyapp/{{ cookiecutter.slug }}/{{ cookiecutter.package_name }}/migrations/env.py

```diff
@@ -1 +0,0 @@
-../../../../_shared/project/migrations/env.py
```

Comment:
Ok, so we're removing migrations from non-Pyramid apps :+1: I remember we discussed this in Slack, because report is the only non-Pyramid app that uses the cookiecutter's Postgres option